.github/workflows/workflow-run-execution-performance.yaml

```diff
@@ -36,15 +41,25 @@ on:
         type: choice
         options:
         - executor-benchmark-runner
+        - benchmark-t2d-32
+        - benchmark-t2d-60
+        - benchmark-c3d-30
+        - benchmark-n4-32
+        - benchmark-c4-32
         description: The name of the runner to use for the test.
-      IS_FULL_RUN:
+      FLOW:
         required: false
-        default: false
-        type: boolean
-        description: Run complete version of the tests
+        default: LAND_BLOCKING
+        options:
+        - LAND_BLOCKING
+        - CONTINUOUS
+        - MAINNET
+        - MAINNET_LARGE_DB
+        type: choice
+        description: Which set of tests to run. MAINNET/MAINNET_LARGE_DB are for performance validation of mainnet nodes.
       IGNORE_TARGET_DETERMINATION:
         required: false
-        default: false
+        default: true
```

Comment:
Are you sure it's a good idea to ignore by default ?

Reply:
this is only for jobs triggered from the UI, right?

if we are triggering from the UI, we really shouldn't be skipping if there are no changes?

Reply:
Yes, you're right.